client/src/app/[site]/globe/components/OpenLayersMap.tsx

```
@@ -0,0 +1,70 @@
+"use client";
+
+import { useRef, useEffect } from "react";
+import Map from "ol/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)